sql/step_74_operation_report_button.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO MENUITEM VALUES ('operationlist','angal.menu.btn.operationlist','angal.menu.operationlist','x','O','printing','org.isf.stat.gui.OperationsListLauncher','N',3);
+INSERT INTO GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('admin','operationlist','Y');

```

Comment:
Wrong (old) value for GM_ACTIVE field

```suggestion
INSERT INTO GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('admin','operationlist','1');

```